average_steer_partitions.py

```diff
@@ -21,6 +21,18 @@ def event_to_analog_value(event: ControlEntry):
     val >>= int32(8)
     return -val
 
+def clean_duplicate_events(events: list):
+    """
+    ESWC events seem a bit janky. Try to get rid of that by removing 'duplicate'
+    events. Take the latest one
+    """
+    events_cleaned = []
+    for i in range(0, (len(events)-1)):
+        if int(events[i+1].time / 10) * 10 != int(events[i].time / 10) * 10:
```

Comment:
What happens when there's events like this?
```
10000 Accelerate (1)
10000 Brake (1)
10010 Accelerate (0)
```
Is the first accelerate simply skipped?

Reply:
Very good point. Reworked the method to skip over everything that is not a `Steer` event